src/v2/view-builder/components/AuthenticatorEnrollOptions.js

```diff
@@ -80,6 +92,36 @@ const AuthenticatorRow = View.extend({
     }
 
     const data = View.prototype.getTemplateData.apply(this, arguments);
+
+    const currentTimestampMs = Date.now();
+    const gracePeriodEpochTimestampMs = new Date(this.model.get('relatesTo')?.gracePeriod?.expiry).getTime();
+    if (!isNaN(gracePeriodEpochTimestampMs) && currentTimestampMs < gracePeriodEpochTimestampMs) {
```

Comment:
Note that `isNaN` coerces the value to a number first, but I think in this particular usage we aren't at risk of non-numeric types being input, since the possible values of `gracePeriodEpochTimestampMs` are either `Number` or `NaN` and no other possibility.